docs/devs/modules.md

```diff
@@ -0,0 +1,79 @@
+# Creating a New Disease Module in Synthea
+
+## Introduction
+This guide will walk you through the steps to create a new disease module in the Synthea synthetic data generator.
+
+## Prerequisites
+- Basic knowledge of JSON
+- Familiarity with Synthea's module framework
+- [NICE guidelines](https://www.nice.org.uk/guidance) for the pathway you want a module for 
+
+## Pre-work 
+We would recommend creating a flow diagram of the NICE pathway, thinking about every condition that would send you down different routes, every situation where you'd interact with the health system (e.g. get a measurement, test, appointment, diagnosis etc.), to base yourself off when it comes to making the module. You can see some example flow charts [here](#example-flowchart).
+
```

Comment:
We recommend starting by creating a flow diagram of the NICE pathway. Consider every condition that might lead to different pathways, and every interaction with the healthcare system (e.g., measurements, tests, appointments, diagnoses, etc.). This will serve as a foundation for developing your module. You can find example flow charts [here](https://github.com/nhsengland/swpc_synthea/pull/32/files#example-flowchart).